vault/import_consul_role_test.go

```diff
@@ -0,0 +1,27 @@
+package vault
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccConsulRole_importBasic(t *testing.T) {
```

Comment:
This test is failing for me locally.
```
--- FAIL: TestAccConsulRole_importBasic (0.01s)
    testing.go:434: Step 0 error: Error applying: 1 error(s) occurred:
        
        * vault_consul_role.test: 1 error(s) occurred:
        
        * vault_consul_role.test: error writing to Vault: Error making API request.
        
        URL: PUT http://localhost:8200/v1/consul/roles/consul--6730473261304153958
        Code: 404. Errors:
        
        * no handler for route 'consul/roles/consul--6730473261304153958'
FAIL
```
Steps to reproduce:
- Run Vault 0.11.1 using `$ vault server -dev`
- Run the test with `$ export TF_ACC=1`, `$ export VAULT_ADDR=http://localhost:8200`, `$ export VAULT_TOKEN=something`